setup.py

```diff
@@ -42,4 +42,5 @@ def get_petsc_dir():
                              include_dirs=include_dirs,
                              extra_link_args=["-L" + link_dir] +
                              ["-Wl,-rpath," + link_dir],
+                             extra_compile_args=["-O3"],
```

Comment:
I agree it's not needed. However, it's nice to have it explicitly here, because if someone wants to compile with debugging symbols it's easy to discover how.